From a725a6c0a74429c490affd351189673646d7ffec Mon Sep 17 00:00:00 2001
From: Stanislav Ochotnicky <sochotnicky@redhat.com>
Date: Mon, 17 May 2010 13:53:18 +0200
Subject: [PATCH 04/10] Remove mvn-release-plugin

for now this is included in current mvn in Fedora. Remove once
release-plugin is separate
---
 pom.xml |   11 -----------
 1 files changed, 0 insertions(+), 11 deletions(-)

diff --git a/pom.xml b/pom.xml
index ae0d95e..f98a700 100644
--- a/pom.xml
+++ b/pom.xml
@@ -351,17 +351,6 @@ target platform and specify -Dntdll_target=msbuild on the mvn command line.
         </executions>
       </plugin>
       <plugin>
-        <artifactId>maven-release-plugin</artifactId>
-        <version>2.0-beta-9</version>
-        <!--
-          Bug MRELEASE-273 has been throwing NPE during release:perform,
-          typically after deploy so this places it last.
-        -->
-        <configuration>
-          <goals>package site-deploy assembly:attached deploy</goals>
-        </configuration>
-      </plugin>
-      <plugin>
         <artifactId>maven-source-plugin</artifactId>
         <version>2.1.1</version>
         <executions>
-- 
1.6.6.1

